### ISSUE TITLE
Won't authorize SSH connection

### ISSUE THREAD

**Issue by selfagency** [ExpandedFronts/Revisr#59]
I created an SSH key for my server and uploaded it to Github.
![screenshot 2015-06-07 17 31 27](https://cloud.githubusercontent.com/assets/2541728/8026185/27d958d6-0d3b-11e5-8be3-29215deaf651.png)

I configured my plugin.
![screenshot 2015-06-07 17 24 23](https://cloud.githubusercontent.com/assets/2541728/8026190/44d81666-0d3b-11e5-81cb-236beb5c7f45.png)

But I'm getting this error and without more detail I can't understand why.
![screenshot 2015-06-07 17 24 54](https://cloud.githubusercontent.com/assets/2541728/8026193/5d0d7050-0d3b-11e5-9079-922417a1da30.png)

**Comment by mgshaw** [comment]
Hi. Can you paste the debug info from the "Help" tab? Are you able to do git push through command line?

**Comment by selfagency** [comment]
I just pushed from the command line and it worked but it asks me for my keyphrase each time. After running it from the command line, I get a new error.

```
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Here's the debug info:

```

Operating System	Linux
Exec() Enabled	true
Git Install Path	/usr/bin/git
Git Version	git version 1.9.2
MySQL Install Path	/usr/bin/mysql
File Permissions	true
Repository Path	/home/blackliv/public_html/cms
Repository Status	Click here to view.
```

**Comment by mgshaw** [comment]
Did you generate the SSH key with a keyphase? If so, you'll need to generate a new key without a keyphase so that it can be used automatically by Revisr.

**Comment by selfagency** [comment]
Ah, I see. Let me try again.

**Comment by selfagency** [comment]
Ok, that worked except now I have a new error.

```
fatal: Couldn't find remote ref master
```

**Comment by mgshaw** [comment]
That's strange. Is there anything on the remote repo? Can you run "git remote -v" in command line and let me know what it outputs?

**Comment by selfagency** [comment]
```
blackliv@serv01 [~/www/cms]# git remote -v
origin	git@github.com:selfagency/client-blacklivesmatterjewishresources.git (fetch)
origin	git@github.com:selfagency/client-blacklivesmatterjewishresources.git (push)
```

**Comment by selfagency** [comment]
The repo is empty. Weird.

**Comment by selfagency** [comment]
Ok. Just did a push from within the WP backend. Now it works. Hooray!

Thanks for your help. Maybe add a line about creating a key with no passphrase in the instructions.

**Comment by mgshaw** [comment]
Awesome, glad it's working! I just added a note about the passphrase to the remote config documentation. 

Marking this as closed but please let me know if you run into any further issues.
